### PR TITLE
Updates to Moderator Helm Chart for EKS migration

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-2165

What this PR does:
* updates ingress to handle k8s api versions beyond 1.14
* reorganizes Values.yaml to fit other patterns, have notes for external-secrets, work with CI setup
* updates some dependencies
* uses cert-manager for moderator TLS certificates (instead of terraform creating ACM certs & attached to route53)
* makes configmap data editable via Values (to support local development injected secrets this way instead of via external-secrets)

Blocking: https://github.com/mozilla-it/itse-apps-stage-1-infra/pull/74

### CODE REVIEW

**Comment by cmharlow** [comment]
Clearly, still need to sort out travis CI issues (this runs fine via ct lint & install locally), and want to add some more details for using this in local dev.

File: charts/mozmoderator/Chart.yaml [hunk 1]
```diff
@@ -2,19 +2,23 @@ apiVersion: v2
...
 keywords:
   - Mozilla
   - moderator
 sources:
   - https://github.com/mozilla/mozmoderator
 maintainers:
-  - name: Alberto del Barrio
-    email: alberto@mozilla.com
+  - name: Christina Harlow
+    email: charlow@mozilla.com
```
[the remark this answers is not in the document]
Comment (cmharlow) [reply charts/mozmoderator/Chart.yaml:14]: do you want to make that PR on all the charts?

[the remark this answers is not in the document]
Comment (cmharlow) [reply charts/mozmoderator/Chart.yaml:14]: also how did this get reviewers, i didn't tag anyone yet

**Comment by cmharlow** [comment]
moving back to draft since it grabbed reviewers somehow when not yet ready.

File: charts/mozmoderator/Chart.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkochendorfer) [reply charts/mozmoderator/Chart.yaml:14]: haha, that is my fault. I opened tabs with all the emails I received didn't notice that this was a draft.

[the remark this answers is not in the document]
Comment (bkochendorfer) [reply charts/mozmoderator/Chart.yaml:14]: I think it makes sense to use our team email and author as we will all probably edit these

**Review by duallain — APPROVED** [review]

**Pull request merged**